### DISCUSSION TITLE
Добавить вариативность поиска преступника.

### DISCUSSION THREAD

**Discussion by Samchikk** [octothorp-team/public-dobrograd#709] [Предложения]
Сейчас все что есть у копов,коронеров и детективов для поиска преступников это материалы и святой дух(рп). Я считаю что нужно расширить спектр улик, которые могут остаться на месте преступления. 
1. Дать возможность класть любую вещь в анализатор и узнавать кто ее держал в руках последний(не считая копов,коронеров,деков) + это пожет еще и в расследовании ограблений.
2. Возможность просматривать все имущество игрока на данный момент(купленные дома, заспавненые машины)
3. Сделать систему отслеживания по камерам(если человека кинули в розыск, то пикает камеры где его видели еще до момента как его подали в розыск(мне это кажется логичным)
4.Дать возможность кидать в розыск транспортные средства.(также с пиликаньем камер)
5. Отрисовывать как либо попадания в тело.

**Comment by travka4000** [comment]
Да я думаю это будет интересно я за
